### PR TITLE
Delete using namespace std in all the files

### PR DESCRIPTION
**Pull request by marcosmarp**
Solves #4

### CODE REVIEW

File: tempCodeRunnerFile.h [hunk 1]
```diff
@@ -0,0 +1 @@
+std::string
```
Comment (nahueespinosa) [inline tempCodeRunnerFile.h:1]: Did you mean to add this new file?

File: main.cpp [hunk 2]
```diff
@@ -1,32 +1,30 @@
...
-			cout << "Invalid number, please, select a valid option (remember, you must enter the number of the option that you want): ";
-			cin >> option;
-			cout << endl;
+			std::cout << "Invalid number, please, select a valid option (remember, you must enter the number of the option that you want): ";
+			std::cin >> option;
+			std::cout << std::endl;
 
 		}
-		/* If the users enters a string of some kind, everything goes crazy, i have to work on that */
+		/* If the users enters a std::string of some kind, everything goes crazy, i have to work on that */
```
Comment (nahueespinosa) [inline main.cpp:27]: Not necessary to edit the comments xD. I'm guessing you used an auto replace feature.

**Review by nahueespinosa — CHANGES_REQUESTED** [review]
It's difficult to compare the changes, as the repository history is kind of messed up.
I think it looks good in general (left some comments).

Also, please untrack `main.exe`.

Might be a good idea to fix the gitignore branch and then fix this.

File: tempCodeRunnerFile.h (hunk 1 above)
Comment (marcosmarp) [reply tempCodeRunnerFile.h:1]: I didn't! I think is a header created by Visual Studio (i switched from Dev C++ to Visual Studio recently). Could it be?

Btw: sorry for the delay in responding, i just got home.

File: main.cpp (hunk 2 above)
Comment (marcosmarp) [reply main.cpp:27]: Lmao I didn't noticed that. Indeed, search and replace

**Comment by marcosmarp** [comment]
Will solve #3 before this one!

**Comment by marcosmarp** [comment]
Now that #3 is solved, can I update this branch so is up to date with the main? Is there a way to do that?

**Comment by nahueespinosa** [comment]
I just rebased the branch with main to make it easier for you. I'll explain how to rebase later!
For now, as your local branch is different from remote (and doesn't share a history anymore), you will need to delete it and fetch the branch again.

```bash
git checkout main
git pull
git branch -D marcos/replace_using_name_space_std        # Delete local branch
git checkout marcos/replace_using_name_space_std         # Fetch and checkout to updated branch
```

Don't forget to address the comments:

- Remove `tempCodeRunnerFile.h`
- Remove `main.exe`
- Fix comment in `main.cpp`

**Comment by marcosmarp** [comment]
Thanks! A question: when I commit the changes it git bash, I think that every change i made share the summary of what I did. Am I using the wrong command? This is what I use 

>$ git add "changedfile1"
$ git add "changedfile2"
$ git commit -m "summary"

This way, i think both files share the summary when I push them to the origin

**Comment by nahueespinosa** [comment]
Yeah, a commit usually includes changes to more than one file.
Take a look at [this page](https://www.atlassian.com/git/tutorials/saving-changes) for more information.

> The git add command adds a change in the working directory to the staging area. It tells Git that you want to include updates to a particular file in the next commit. However, git add doesn't really affect the repository in any significant way—changes are not actually recorded until you run git commit.

**Comment by marcosmarp** [comment]
Just read it! There's one thing I didn't quite understood. 

What if i modify different files for different reasons? How do I comment in each one what i did when commiting and pushing?

For example, what you commented earlier. I didn't addressed each particular change on each file, they all just were pushed with the "deleted using namespace std" summary

**Comment by nahueespinosa** [comment]
You would do something like:

```bash
# Changes some files
$ git add "changedfile1"
$ git add "changedfile2"
$ git commit -m "changes to implement part of my work"
# Changes other files
$ git add "changedfile2"
$ git add "changedfile3"
$ git commit -m "changes to implement a different part of my work"
# More changes
$ git add "changedfile1"
$ git commit -m "last small fix to the previous commited code"
```

Then you'll end up with a history of 3 commits reflecting your work in your feature branch. Of course "Squash and merge" will still squash them all in one single commit representing the whole feature.

**Comment by marcosmarp** [comment]
Got it! Thanks.

Just deleted the local branch and fetched again. .exe seems to be running just fine, and the .gitignore is there. Should I do anything else before merging?

**Comment by nahueespinosa** [comment]
Yes!

- Remove `tempCodeRunnerFile.h`
- Remove `main.exe`
- Fix comment in `main.cpp`

Commit those changes and push them.

**Comment by marcosmarp** [comment]
Just done it! Sorry, forgot about doing those 3 changes.

**Review by nahueespinosa — APPROVED** [review]
Approved! Great

**Pull request merged**